awkward/pandas/accessor.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+from pandas.api.extensions import ExtensionDtype
+
+def delegated_method(method, index, name, *args, **kwargs):
+    return pd.Series
+
+import awkward.array
+
+class Delegated:
```

Comment:
`Deligated(object)` for Python 2 niceness. (Needs to subclass `object` to be a new-style class.)

Reply:
Done.